src/PrestaShopBundle/Install/Install.php

```diff
@@ -848,6 +848,7 @@ public function configureShop(array $data = [])
 
                 return false;
             }
+            Context::getContext()->employee = $employee;
```

Comment:
Please use `use PrestaShop\PrestaShop\Adapter\LegacyContext;`

Reply:
The whole `Install` class is using `Core` classes for `Context`, `Configuration` or even `Tools`. I’m not sure we should mix `Adapter` classes and `Core` classes (and refactoring all of this would be out of scope). WDYT?

Reply:
Approved. I forgot it's in an Install namespace.